CHANGELOG.md

```diff
@@ -14,7 +14,10 @@
 * New `Badge` component allows a styled badge to be placed inline with text/title, e.g. to show a
   counter or status indicator within a tab title or menu item.
 * Updated `TreeMap` color scheme, with a dedicated set of colors for dark mode.
-
+* New XH convenience methods `successToast()`, `warningToast()`, and `dangerToast()` show toast
+  alerts with matching intents and appropriate icons.
+  * ⚠ Note that the default `XH.toast()` call now shows a toast with the primary (blue) intent and
```

Comment:
Strictly speaking it's a breaking change, right?  If the UI was being tested with a before/after image comparer, it would say the toast broke.  Maybe a little note under the breaking changes section?  Can't do any harm.

Reply:
we don't consider look and feel changes like this breaking.  If we did, every change would be a major version....